openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -806,6 +806,28 @@ def match_volume_by_prefix(self, prefix):
         match_heketi_and_gluster_volumes_by_prefix(
             h_vol_names, "{}_".format(prefix))
 
+    def cleanup_heketi_volumes(
```

Comment:
Cleanup for heketi file volumes are not need, make this function specific to delete blockhosting volume and change name accordinlgy.

Reply:
Done